autoload/neomake/makers/ft/swift.vim

```diff
@@ -41,6 +49,26 @@ function! neomake#makers#ft#swift#swiftpmtest() abort
     return maker
 endfunction
 
+function! neomake#makers#ft#swift#swiftfile() abort
+    let maker = s:get_swiftpm_base_maker()
+    let maker.append_file = 1
+    return maker
+endfunction
+
+function! neomake#makers#ft#swift#swiftlint() abort
+    return {
+        \ 'args': ['lint'],
+        \ 'append_file': 1,
+        \ 'errorformat':
+            \ '%E%f:%l:%c: error: %m,' .
+            \ '%W%f:%l:%c: warning: %m,' .
+            \ '%E%f:%l: error: %m,' .
+            \ '%W%f:%l: warning: %m,' .
+            \ '%Z%\s%#^~%#,' .
+            \ '%-G%.%#',
+        \ }
```

Comment:
This appears to slightly extend (and maybe therefore only fix) the other errorformats being used.

Would be good to have some example output to test this with, and merge it with the base efm eventually.

Reply:
```
Linting Swift files at paths /Users/jandamm/Formula.swift
Linting 'Formula.swift' (1/1)
/Users/jandamm/Formula.swift:169:15: error: Force Cast Violation: Force casts should be avoided. (force_cast)
/Users/jandamm/Formula.swift:1:15: error: Identifier Name Violation: Variable name should be between 3 and 40 characters long: 'f' (identifier_name)
/Users/jandamm/Formula.swift:78:16: error: Identifier Name Violation: Variable name should be between 3 and 40 characters long: 's' (identifier_name)
/Users/jandamm/Formula.swift:143:1: warning: Identifier Name Violation: Variable name should be between 3 and 40 characters long: 'f1' (identifier_name)
/Users/jandamm/Formula.swift:144:1: warning: Identifier Name Violation: Variable name should be between 3 and 40 characters long: 'f2' (identifier_name)
/Users/jandamm/Formula.swift:91:1: warning: Line Length Violation: Line should be 120 characters or less: currently 133 characters (line_length)
/Users/jandamm/Formula.swift:137:4: warning: Switch and Case Statement Alignment Violation: Case statements should vertically align with their enclosing switch statement. (switch_case_alignment)
/Users/jandamm/Formula.swift:133:1: warning: Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 2. (vertical_whitespace)
Done linting! Found 8 violations, 3 serious in 1 file.
```

I've seen [here](https://github.com/keith/swift.vim/blob/master/syntax_checkers/swift/swiftlint.vim#L24) that sometimes the columns seem to be missing.
Without the last two lines all lines are shown in.

Reply:
I've changed the error format. Should be good now 👍